packages/api-client/src/user/UserAPI.ts

```diff
@@ -385,6 +385,35 @@ export class UserAPI {
     return response.data;
   }
 
+  /**
+   * Check availability of a single user handle.
+   * @param handle The handle to check
+   * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/users/checkUserHandle
+   */
+  public async headHandle(handle: string): Promise<void> {
+    const config: AxiosRequestConfig = {
+      method: 'head',
+      url: `${UserAPI.URL.USERS}/${UserAPI.URL.HANDLES}/${handle}`,
+    };
+
+    await this.client.sendJSON<string[]>(config);
```

Comment:
`string[]` is not needed

```suggestion
    await this.client.sendJSON(config);
```